bench/tx-generator/README.md

```diff
@@ -0,0 +1 @@
+# Transaction Generator
```

Comment:
Can we add an introduction as to how to use the transaction generator and list the team responsible for this library?